app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
```

Comment:
The table name 'actors' is correctly in plural form, which aligns with the checklist requirement that if a database contains more than one object, it should be named in plural.